WooCommerce/Classes/ViewModels/Order Details/Aggregate Order Items/AggregateOrderItem.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+
+/// This model represents a computed summary of order items.
+/// (order items - refunded order items) = aggregate order item data.
+///
+final class AggregateOrderItem {
```

Comment:
This is a class because I planned to attach a new property in a "helper method" extension. Added it to this PR.

Reply:
another suggestion: might be worth mentioning "Product" in the name, since it's representing a Product related to an Order

Reply:
Structs can't have properties in extensions. So I could change this to a struct and add the property to the top. I don't have a good reason other than code organization, since the other things I was experimenting with don't exist anymore 😀